layouts/partials/css.html

```diff
@@ -21,34 +21,33 @@
 <!-- Fallback font for symbols (such as "🛈"). --
 <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Noto+Sans+Symbols+2">
 
-<!-- SASS -->
-{{- $sass := (slice "theme-css/pst/pydata-sphinx-theme.scss") | append (resources.Match "css/*.scss") -}}
+<!-- Process and include Sass files. -->
+{{- $sass := (slice "theme-css/pst/pydata-sphinx-theme.scss")
+           | append (resources.Match "theme-css/*.scss")
```

Comment:
`theme-css/**/*.scss`?

Reply:
or `*-css/**/*.scss`, then you avoid the empty slices too.

Reply:
I'm not sure, but I've spun out that commit to https://github.com/scientific-python/scientific-python-hugo-theme/pull/429.